delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/organizations/OrganizationState.scala

```diff
@@ -0,0 +1,97 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.organizations
+
+import java.time.Instant
+import java.util.UUID
+
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.{nxv, schemas}
+import ch.epfl.bluebrain.nexus.delta.sdk.OrganizationResource
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceRef.Latest
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{Label, ResourceF, ResourceRef}
+import org.apache.jena.iri.IRI
+
+/**
+  * Enumeration of organization states.
+  */
+sealed trait OrganizationState extends Product with Serializable {
+
```

Comment:
add label, uuid and deprecated

Reply:
There is no label, uuid or deprecation flag for Initial state. So how this is helpful? I would have to add
```
def label: Option[Label]
def uuid: Option[UUID]
```

and for what is useful?